sdks/python/apache_beam/internal/gcp/auth.py

```diff
@@ -125,30 +139,22 @@ def get_service_credentials(cls):
           "socket default timeout is %s seconds.", socket.getdefaulttimeout())
 
       cls._credentials = cls._get_service_credentials()
+      cls._add_impersonation_credentials()
       cls._credentials_init = True
 
     return cls._credentials
 
   @staticmethod
-  def _get_service_credentials():
+  def _get_service_credentials(cls):
     if not _GOOGLE_AUTH_AVAILABLE:
       _LOGGER.warning(
           'Unable to find default credentials because the google-auth library '
           'is not available. Install the gcp extra (apache_beam[gcp]) to use '
           'Google default credentials. Connecting anonymously.')
       return None
 
-    client_scopes = [
-        'https://www.googleapis.com/auth/bigquery',
```

Comment:
CC: @chamikaramj 
Would it make sense to combine the scopes in Java (GcpCredentialFactory.java) and in Python in a properties file or proto enum so that XLang will have the same scopes irregardless of whether the Pipeline is constructed using Python or Java. 

We can combine the two lists together. Feel free top create a different JIRA about this and defer to later.

Reply:
+1. That will make things much more consistent for x-lang (and users of different SDKs in general).